tests/middleware/test_base.py

```diff
@@ -6,47 +6,46 @@
 from starlette.testclient import TestClient
 
 
-class CustomMiddleware(BaseHTTPMiddleware):
-    async def dispatch(self, request, call_next):
-        response = await call_next(request)
-        response.headers["Custom-Header"] = "Example"
-        return response
-
+def test_custom_middleware():
+    class CustomMiddleware(BaseHTTPMiddleware):
+        async def dispatch(self, request, call_next):
+            response = await call_next(request)
+            response.headers["Custom-Header"] = "Example"
+            return response
```

Comment:
It'd be better to leave all these bits as they were - it's super important that we always try to keep the footprint of a pull request absoultely minimal, and keep it scoped to one thing only.